### DISCUSSION TITLE
OpacityMaps

### DISCUSSION THREAD

**Discussion by almostEric** [CommunityToolkit/Maui#1048] [New Feature Discussions]
Can we have OpacityMaps like WPF XAML has?

**Comment by bijington** [comment]
Do you mean OpacityMasks? If so I saw a tweet the other day mentioning it isn't in .NET MAUI yet. https://twitter.com/tweet2anees/status/1627948291450351616

I haven't checked through the MAUI repo but that would be the best place to request the feature.

**Comment by CliffAgius** [comment]
Closed and answered
